autotest.pm

```diff
@@ -360,6 +360,10 @@ sub runalltests {
                 }
             }
             else {
+                if(!$flags->{norollback} && $last_milestone && $flags->{force_reset}) {
```

Comment:
Could you please test (locally, manually, once) what happens when you execute this on non-qemu backends?

Reply:
Here we go: http://g226.suse.de/tests/2666#step/openvswitch/4 
So test failed, as file was still there (no revert done), no error in the logs.